overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -18651,7 +18651,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§9Phosphorus§r, §9Oxygen§r, and §9Water§r combine together into §9Phosphoric Acid§r.",
+          "desc:8": "§9Phosphorus§r, §9Oxygen§r, and §9Water§r combine together into §9Phosphoric Acid§r.\n\nAlternatively, you can combine §P§6Phosphorous Pentoxide§r and §WWater§r for a much faster reaction.",
```

Comment:
Please double-check the formatting codes on this one, it looks like some extra stuff got in there.